contrib/opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -364,6 +403,7 @@ def _transcribe(item, parents, editRate, masterMobs):
         result.out_offset = otio.opentime.RationalTime(out_offset, editRate)
 
     elif isinstance(item, aaf2.components.Filler):
+        transcribe_log("Creating Gap for {}".format(_encoded_name(item)), indent)
```

Comment:
If we're going to include these `trancribe_log` calls for each type, I'd rather we use a consistent format, so:

`transcribe_log("Gap", item, indent)`

...and then inside `transcribe_log()` it unrolls the name into the message `"Creating {} for {}".format(typestr, _encoded_name(item)), indent)`

...but the broader problem with these kinds of logs from adapters is that we don't have a consistent framework for it.  @reinecke just flagging this since I know its something you've thought about in the past.
and then have transcribe_log format the item name and string consistently (unless there is a lot of helpful variation in these strings)

Reply:
yeah, good point. It started as a more generic log, but since it's now specific to the _transcribe() method, this is a good idea.

Reply:
This is an adapter argument now - does that satisfy your concern @ssteinbach ?